### PR TITLE
Soc 1508 && SOC-1509 "More" icon on posts and replies

### PR DESCRIPTION
**Pull request by bkoval**
https://wikia-inc.atlassian.net/browse/SOC-1508
https://wikia-inc.atlassian.net/browse/SOC-1509

@katanka @rafalkalinski @kvas-damian @Bloodmoon24

### CODE REVIEW

File: front/main/app/templates/components/post-detail.hbs [hunk 1]
```diff
@@ -66,8 +64,8 @@
 </div>
 {{#if post.isLoading}}
 	{{loading-spinner
-	active=post.isLoading
-	radius=32
-	strokeWidth=2
+		active=post.isLoading
```
Comment (rogatty) [inline front/main/app/templates/components/post-detail.hbs]: Needs more `\t`

Comment (bkoval) [inline front/main/app/templates/components/post-detail.hbs]: Are you sure?

File: front/main/app/templates/components/post-reply.hbs [hunk 2]
```diff
@@ -14,11 +14,9 @@
 		<span class="timestamp">
 			&bull; {{time-ago post.creationDate.epochSecond}}
 		</span>
-		{{discussion-delete
-			delete=this.attrs.deleteReply
-			undelete=this.attrs.undeleteReply
-			post=post
-		}}
+		<div class="discussion-post-more-options">
```
Comment (kvas-damian) [inline front/main/app/templates/components/post-reply.hbs]: It's reply, so class name is misleading

File: front/main/app/templates/components/post-detail.hbs (hunk 1 above)
Comment (rogatty) [inline front/main/app/templates/components/post-detail.hbs]: Yes, I am.

**Pull request closed without merging**